repose-aggregator/core/repose-core/src/main/java/org/openrepose/powerfilter/PowerFilter.java

```diff
@@ -390,9 +389,11 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
         try {
             context = tracer.extract(Format.Builtin.HTTP_HEADERS, new TracerExtractor(wrappedRequest));
         } catch (RuntimeException re) {
-            LOG.error("Incoming tracer could not be parsed.");
-            LOG.error("We're going to start a new root span here; even though this is most likely part of a larger span.");
-            LOG.error("Check out following exception for more details:", re);
+            LOG.error("%s %s %s",
```

Comment:
Huh, does the `%s` pattern work here, or do you have to use `{}`?

Reply:
Yep. Fixed.